nonce/nonce.go

```diff
@@ -117,11 +138,25 @@ func (ns *NonceService) encrypt(counter int64) (string, error) {
 	ct := ns.gcm.Seal(nil, nonce, pt, nil)
 	copy(ret, nonce[4:])
 	copy(ret[8:], ct)
-	return base64.RawURLEncoding.EncodeToString(ret), nil
+
+	return ns.prefix + base64.RawURLEncoding.EncodeToString(ret), nil
 }
 
 func (ns *NonceService) decrypt(nonce string) (int64, error) {
-	decoded, err := base64.RawURLEncoding.DecodeString(nonce)
+	var decoded []byte
```

Comment:
This can be declared at the same time you call `DecodeString`; no need to declare it up here.